pkg/lua/Makefile

```diff
@@ -1,11 +1,11 @@
 PKG_NAME=lua
 PKG_URL=https://github.com/lua/lua.git
-PKG_VERSION=e354c6355e7f48e087678ec49e340ca0696725b1
+PKG_VERSION=v5-3-4
```

Comment:
is this v5-3-4 a git identifier?

Reply:
Yes, it is a tag.

Reply:
ok!

Reply:
This has gone back to a commit hash, because tags can change.